### DISCUSSION TITLE
PaginationState

### DISCUSSION THREAD

**Discussion by Ali-Hussein-dev** [TanStack/table#4337] [General] [answered]
I want to customize the default page size so I did that 
```jsx
const table= useReactTable({
//...
state: {
      pagination: { pageSize: 20, pageIndex: 0 },
    },
})
// ...
```
but the next page and previous page buttons stop working. What happened?

**Comment by mukul-vm** [comment, marked as answer]
This is 

> I want to customize the default page size so I did that
> 
> ```js
> const table= useReactTable({
> //...
> state: {
>       pagination: { pageSize: 20, pageIndex: 0 },
>     },
> })
> // ...
> ```
> 
> but the next page and previous page buttons stop working. What happened?

This is because you are fixing the state of the table to pageSize: 0 and pageIndex: 0. Instead of _state_ you should use _initialState_.
e.g. 
`initialState: {
    pagination: { pageSize: 20, pageIndex: 0 }
}`
